Theta Project Update v1.1 for New Game Mode

lngredients opened the pull request:
- Changed Theta Project Armor to a cooler looking armor set.
- Theta armor should be capable of holding most weapons in its suit slot.
- Gave the theta project closet a bullpup rifle and three mags to equip their creation, or be killed by it.
- The armor gives a slight boost to walk speed.
- Theta Undersuit was replaced and now will actually work.
- Slightly Changed Theta Project equipment names so they arn't so long.
- Deleted Theta Boots since they are redundant
- Decreased armor in Theta Arm Guards

lngredients committed:
Theta Project Update

Changed Theta Project Armor to a cooler looking armor set.
Theta armor should be capable of holding most weapons in its suit slot.
Gave the theta project closet a bullpup rifle and three mags to equip their creation, or be killed by it.
The armor gives a slight boost to walk speed.
Theta Undersuit was replaced and now will actually work.
Slightly Changed Theta Project equipment names so they arn't so long.
Deleted Theta Boots since they are redundant
Decreased armor in Theta Arm Guards

lngredients commented:
Don't merge this just yet.

lngredients committed:
Theta Project v1.2

Made armor thicker on Theta armor to facilitate longer usage time.
Added back in Theta Boots.
5 armor thickness to boots and gloves.
10 armor thickness on Theta Undersuit.
Minimum breath pressure halved for augmented lungs.
Slightly increased lung max damage because it was formally at the native damage levels.
Added descriptions to all the augmented organs.

lngredients commented:
If the CI passes this is now ok to merge.

lngredients commented:
How do I add a change log?

bloxgate commented:
Take a look at  html/changelogs/__CHANGELOG_README.txt 

It's really simple.

lngredients committed:
Adding in Changelog to Edit

lngredients committed:
Updated the change log, I think it should be good.

: 3

lngredients commented:
Let me know if that is good Blox.

lngredients commented:
I would love for this to be reviewed if possible.

caelaislinn commented:
Map changes are a little more difficult to review. I'll have a look now

lngredients commented:
Thanks Cael.

lngredients commented:
Line 642 should contain all the relevant changes from this PR I believe.

caelaislinn commented:
Maps aren't reviewed by looking at the text. We download the map and go through it in the editor

lngredients committed:
Fixed Typo in Theta Equipment

Theta Armor -Was "Feat" changed to "Fear"
Theta Undersuit- Was "Progam" changed to "Program"
Theta Boots-"Inlaces" changed to "Inlaced"
Decreased Theta Boots armor values to be more in line with the gloves.

lngredients commented:
Fair enough, I noticed some typos and other things so I am pushing a little update.

lngredients committed:
Theta Boots Armor Value Fix.

Decreased armor from Theta Boots to be more in line with the Arm Guards.

caelaislinn reviewed (dismissed):
I like what you've done here, these seem to be some good balance changes. They need to be done in the code though, not on the map. The changelog has too many entries for the limited number of changes as well.

lngredients commented:
Well I'm not quite well versed enough to create entirely new objects through the code. I used variable changes because I'm more familiar with how they work, and it takes less time then parsing code.

lngredients commented:
Is there a downside to doing them on the map?

caelaislinn commented:
It's hard for maintainers to review, and its hard for anyone to make further balance tweaks later.

lngredients commented:
Blurgh, well then how would I go about making these items in the code?

caelaislinn commented:
All of the Theta gear appears to be pre-existing items except for 2/4 of the organs. Create a new subtype of the pre-existing weapons and armour in a new file somewhere inside code/modules/halo/, then change your variables to match the values you want set.

DoctorOctopusTheOctopi commented on html/changelogs/Ingredients-Theta Project Upgrades.yml:
```diff
@@ -0,0 +1,49 @@
+  - tweak: "10 armor thickness on Theta Undersuit."
+  - tweak: "Minimum breath pressure halved for augmented lungs."
+  - tweak: "Slightly increased lung max damage because it was formally at the native damage levels"
+  - tweak: "Added descriptions to all the augmented organs."
```
Should be shrunk down. You can smash all things that relate together, "decreased armor in Theta Arm Guards." can smash the "Decreased armor from theta boots" in with the theta arm guards.. For example
_"Decreased armor in Theta Arm Guards and boots."_  
You can remove the 5 armor thickness and 10 armor thickness lines from the changelog, people will get confused.

lngredients committed:
Update Ingredients-Theta Project Upgrades.yml

lngredients committed:
Condensed Logs

lngredients committed:
Added Theta Objects and Organs into Code

lngredients committed:
Merge branch 'Theta-Project-Upgrades' of https://github.com/lngredients/HaloSpaceStation13 into Theta-Project-Upgrades

lngredients committed:
Carbine Item state change

lngredients commented:
Apparently I don't have the newest version of Python? Is that what is going on?

lngredients commented:
I don't understand why the CI is no longer passing.

lngredients committed:
Armor thickness change

lngredients committed:
Changelog

lngredients committed:
Missing a quotation mark

lngredients committed:
Changing native wielded state.

lngredients commented:
Once this last CI is finished this PR should be good to go.

lngredients commented:
It is done! I tested it and the new gun icon seems to be working even while wielded. All the armor is well in place and the organs are on point, though I can't really put them inside myself they are functionally the same as before. Someone for the love of all that is Halo review my PR! *sobs*

BDpuffy420 reviewed (approved):
lgtm

caelaislinn reviewed (approved):
Looks fine. Still think there's far too many changelog entries. You don't need 13 entries for balance tweaks.
